### PR TITLE
82 export report into html

### PR DESCRIPTION
**Pull request by tonyshumlh**
Please have the report HTML format v1 below.
I personally think that `line number` might not be necessary/good-to-have given the `file path` and `function names` are provided

Screenshot 2024-05-21 at 2 09 58 PM

### CODE REVIEW

**Review by SoloSynth1 — COMMENTED** [review]
@tonyshumlh Have you checked can we first transform the report into a pivot table, then output into HTML? I think the table is a bit hard to parse by eyes right now.

**Comment by tonyshumlh** [comment]
@SoloSynth1 I just had a discussion with @JohnShiuMK on the design of the report to be more human-readable. Great if you can comment on the design draft below. At the meantime, I will take ref of your checklist export function using Pandoc 
IMG_8995

**Comment by tonyshumlh** [comment]
The latest format 
Screenshot 2024-05-22 at 3 39 35 PM
Screenshot 2024-05-22 at 3 39 47 PM

**Review by jinyz8888 — COMMENTED** [review]
@tonyshumlh and @SoloSynth1 . Here's the error message I received:
.../test_creation/modules/checklist/checklist.py", line 116, in read
    raise SyntaxError("Failed to parse the checklist. Make sure that it is a directory containing CSV files.")
SyntaxError: Failed to parse the checklist. Make sure that it is a directory containing CSV files.

which checklist file do you used?

**Comment by jinyz8888** [comment]
@tonyshumlh Can't find the output report, it is not in report/ nor docs/. Where is it?

**Comment by tonyshumlh** [comment]
To be closed after https://github.com/UBC-MDS/test-creation/pull/95 is closed
